espq.py

```diff
@@ -481,11 +485,28 @@ def choose_devices(devices):
 
     choice_list = []
     current_type = None
+    name_len_spacer = max([len(dev.name) for dev in devices]) + 1
+
+    if query == True:
+        print('Querying all devices for version numbers. This make take a minute...')
+        choice_list.append(Separator('Name'.center(name_len_spacer) + 'Firmware'.center(15) + 'Core'.center(7) + 'State'))
+
     for device in devices:
```

Comment:
This expects devices to be sorted.
Should we sort it here?

Reply:
They're already sorted in import_devices()